service/kas/kas.proto

```diff
@@ -31,6 +31,38 @@ message LegacyPublicKeyRequest {
   string algorithm = 1;
 }
 
+message KeyAccess {
+  string encrypted_metadata = 1;
+  bytes policy_binding = 2;
```

Comment:
I'd say this should be an inner type PolicyBinding to match the TDF spec for KAOs, but in the **Request Body Example** it is a string? So IDK.

https://github.com/opentdf/spec/blob/main/schema/tdf/KeyAccessObject.md

Reply:
Created PolicyBinding message to match spec.